dev/src/glvr.cpp

```diff
@@ -181,7 +181,7 @@ void VREye(int eye, float znear, float zfar) {
     SwitchToFrameBuffer(mstex[eye], GetScreenSize(), true, mstf, retex[eye]);
     auto proj =
         FromOpenVR(vrsys->GetProjectionMatrix((vr::EVREye)eye, znear, zfar));
-    Set3DMode(80, 1, znear, zfar);
+    Set3DMode(80, int2_1, znear, zfar);
```

Comment:
this seems to be the hacky part.. instead of an aspect ratio of `1` were now passing pixel size of `(1,1)` as magical value? Why not pass `rtsize` and remove the special case if-check above for the same result?